src/main/java/Duke.java

```diff
@@ -1,10 +1,230 @@
+        FileWriter fw = new FileWriter(filePath);
+        int i = 0;
+        while(i < taskList.size()) {
+            fw.write(taskList.get(i).saveToFile() + System.lineSeparator());
+            i += 1;
+        }
+        fw.close();
+    }
+
+    /**
+     * Reads content of the file and stores the tasks into taskList to be used by the user.
+     *
+     * @param filePath Path to the file that is read from
+     */
+    private static void readFileContents(String filePath) {
+        File f = new File(filePath);
+        try {
+            if (!f.createNewFile()) {
+                Scanner s = new Scanner(f); // create a Scanner using the File as the source
+                while (s.hasNext()) {
+                    String line = s.nextLine();
+                    if (line.startsWith("E")) {
+                        int startDate = line.indexOf('|');
+                        String description = line.substring(4, startDate - 1);
+                        String date = line.substring(startDate + 2);
+                        Event newTask = new Event(description, date);
+                        taskList.add(newTask);
+                        if (line.substring(2).startsWith("1")) {
+                            taskList.get(taskList.size() - 1).setDone();
+                        }
+                    } else if (line.startsWith("D")) {
+                        int startDate = line.indexOf('|');
+                        String description = line.substring(4, startDate - 1);
+                        String date = line.substring(startDate + 2);
+                        Deadline newTask = new Deadline(description, date);
+                        taskList.add(newTask);
+                        if (line.substring(2).startsWith("1")) {
+                            taskList.get(taskList.size() - 1).setDone();
+                        }
+                    } else {
+                        String description = line.substring(4);
+                        Todo newTask = new Todo(description);
+                        taskList.add(newTask);
+                        if (line.substring(2).startsWith("1")) {
+                            taskList.get(taskList.size() - 1).setDone();
+                        }
+                    }
+                }
+            }
+        } catch (Exception e) {
+            System.out.println("sth");
+        }
+    }
+
+
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        readFileContents("lines.txt");
+
+        Scanner in = new Scanner(System.in);
+
+        System.out.println("Hello Bbygirl! I'm Your Boyfriend <3");
+        System.out.println("How can I help you today? ;)");
+        String t = in.nextLine();
```

Comment:
Consider using a more meaningful variable name such as `input` to improve the readability of your code.